SPECS-EXTENDED/flac/flac.spec

```diff
@@ -41,8 +35,8 @@ various music players.
 This package contains the command-line tools and documentation.
 
 %package libs
-Summary: Libraries for the Free Lossless Audio Codec
-Obsoletes: flac < 1.2.1-11
+Summary:        Libraries for the Free Lossless Audio Codec
+Obsoletes:      flac < 1.2.1-11
```

Comment:
I don't think we even have the older flac.  We can probably delete this line entirely

Reply:
Sure